lib/record_store/provider/dnsimple/patch_api_header.rb

```diff
@@ -19,12 +19,17 @@ def execute(method, path, data = nil, options = {})
 
     private
 
-    def rate_limit_sleep(rate_limit_reset, rate_limit_remaining)
+    def rate_limit_sleep(_rate_limit_reset, _rate_limit_remaining)
```

Comment:
Why change the variable names ? Seems those are not used anymore here ?
Where does the value `rate_limit_reset` come from in the next line ?

Reply:
oops i think this might have been a rubocop thing, have it set to format on save, and was playing around with the rubocop configs haha. I'll fix this.